examples/gameserver.yaml

```diff
@@ -78,33 +78,33 @@ spec:
     # conflict with other TCP connections.
     grpcPort: 9357
     httpPort: 9358
-  # [Stage:Alpha]
-  # [FeatureFlag:PlayerTracking]
-  # Players provides the configuration for player tracking features.
-  # Commented out since Alpha, and disabled by default
-  # players:
-  #  # set this GameServer's initial player capacity
-  #   initialCapacity: 10
-  #
-  # [Stage:Alpha]
-  # [FeatureFlag:CountsAndLists]
-  # Counts and Lists provides the configuration for generic (player, room, session, etc.) tracking features.
-  # Commented out since Alpha, and disabled by default
-  # counters: # counters are int64 counters that can be incremented and decremented by set amounts. Keys must be declared at GameServer creation time.
-  #   games: # arbitrary key.
-  #     count: 1 # initial value.
-  #     capacity: 100 # (Optional) Defaults to 1000 and setting capacity to max(int64) may lead to issues and is not recommended. See GitHub issue https://github.com/googleforgames/agones/issues/3636 for more details.
-  #   sessions:
-  #     count: 1
-  # lists: # lists are lists of values stored against this GameServer that can be added and deleted from. Keys must be declared at GameServer creation time.
-  #   players: # an empty list, with a capacity set to 10.
-  #     capacity: 10  # capacity value, defaults to 1000.
-  #   rooms:
-  #     capacity: 333
-  #     values: # initial set of values in a list.
-  #       - room1
-  #       - room2
-  #       - room3
+    # [Stage:Alpha]
+    # [FeatureFlag:PlayerTracking]
+    # Players provides the configuration for player tracking features.
+    # Commented out since Alpha, and disabled by default
+    # players:
+    #  # set this GameServer's initial player capacity
+    #   initialCapacity: 10
+    #
+    # [Stage:Beta]
+    # [FeatureFlag:CountsAndLists]
+    # Counts and Lists provides the configuration for generic (player, room, session, etc.) tracking features.
+    # Now in Beta, and enabled by default
+    counters:  # counters are int64 counters that can be incremented and decremented by set amounts. Keys must be declared at GameServer creation time.
+      games:  # arbitrary key.
+        count: 1  # initial value.
+        capacity: 100  # (Optional) Defaults to 1000 and setting capacity to max(int64) may lead to issues and is not recommended. See GitHub issue https://github.com/googleforgames/agones/issues/3636 for more details.
+      sessions:
+        count: 1
+    lists:  # lists are lists of values stored against this GameServer that can be added and deleted from. Keys must be declared at GameServer creation time.
+      players:  # an empty list, with a capacity set to 10.
+        capacity: 10  # capacity value, defaults to 1000.
+      rooms:
+        capacity: 333
+        values:  # initial set of values in a list.
+          - room1
+          - room2
+          - room3
```

Comment:
Shouldn't that be at the `spec` level instead (one level below) ? It seems to be added at the sdkServer level here 
https://agones.dev/site/docs/reference/agones_crd_api_reference/#agones.dev/v1.GameServerSpec

Reply:
Thanks @lacroixthomas! I've fixed the alignment for counters and lists.